File: examples/send-q.py
# In this part, I'd like to send to the same ip router as far as I can ... when time elapses 

from time import sleep
import mysocket 

me = mysocket.MySocket() 
me.bind(("127.6.6.6", 6666))

print ("sender binds successfully ")

time_wait = 5 
while True: 
    print (f"wait for {time_wait} s, then send a packet with 1 byte (80)")
    sleep(time_wait) 
    time_wait *= 0.95 
    if time_wait < 0.1 : 
        time_wait = 0.1 
    me.sendto(b"\x80", ("127.6.6.6", 6665)) 

File: examples/send-q.rs
use std::net::UdpSocket;

// Example 1: send and receive in the same router: 127.6.6.6 (6666 -> 6665) 
// sender: send-q.py 
// recvive: recv-q.py 

fn main() {
    let sender = UdpSocket::bind("127.32.68.101:54528").unwrap(); 
    eprintln!("run the controller for 'q' target. "); 
    sender.send_to("ROUTER 127.6.6.6".as_bytes(), "127.67.117.116:52736").unwrap(); 
}